static/js/components/forms/SiteContentForm.tsx

```diff
@@ -37,21 +38,23 @@ interface Props {
   configItem: EditableConfigItem
   content: WebsiteContent | null
   formType: ContentFormType
+  website: Website
 }
 
 export default function SiteContentForm({
   onSubmit,
   fields,
   configItem,
   content,
-  formType
+  formType,
+  website
```

Comment:
if you want you could use the `useWebsite` hook here instead of passing in a prop

Reply:
It works fine but I think we would need to rewrite many of the tests in `SiteContentForm.test.tsx` because it would require `helper.configureRenderer`. I think some things like whether `isSubmitting` is disabling the button become impossible to test since we would not have access to the inner function anymore

Reply:
hmm what if we did a `jest.mock` on `useWebsite` and just made it return a website we made with out website factory?

Reply:
possibly that would let us still use the shallow renderer

Reply:
That's an interesting idea, I'll try that

Reply:
It worked out, I updated the tests for it